### ISSUE TITLE
wrong data in gitpod usage

### ISSUE THREAD

**Issue by elaihau** [gitpod-io/gitpod#636]
**Describe the bug**

I run out of 100 free hours in 1 hour

![Screenshot from 2019-06-25 08-47-03](https://user-images.githubusercontent.com/37082801/60099607-003c1600-9726-11e9-9698-c4464d8df074.png)

**To Reproduce**

The only thing I did today (so far) was force-pushing a change to update my pull request.

**Comment by vince-fugnitto** [comment]
I experienced something similar as well, I see odd usage on my end and it looks like my 100 hours are up already too.

<div align='center'>


![image](https://user-images.githubusercontent.com/40359487/60100426-b81df300-9727-11e9-9d17-d9aafc72f232.png)

</div>

**Comment by meysholdt** [comment]
hi @elaihau, hi @vince-fugnitto, to unblock you I've increased your monthly hours. We'll investigate and fix this and once fixed, reduce your monthly hours to the regular amount of your subscription. I'm sorry for the trouble and thank you very much for bringing this to our attention.

**Comment by vince-fugnitto** [comment]
Thank you @meysholdt, glad we could help :)

**Comment by rebwill** [comment]
Hey team, I'm having a similar problem. 

Today, Aug. 2nd, I was suddenly locked out of all my Gitpod projects with the message "Remaining usage time: 0:00:00" appearing at the top of my dashboard. (I'm a free user with 100 hours/month of usage).

Two problems with this:
1) It's only the 2nd of the month so there's no way I used 100 hours. Between today and yesterday I used *MAYBE* 10 hrs.
2) Even if it were true, I received absolutely no warning/countdown, so I was unable to rescue any of my projects before being locked out. 

To reproduce: 
I was running two workspaces in the morning. Took a break of a couple of hours, went to start them again, and received this message.
